Dockerfiles/agent/README.md

```diff
@@ -30,3 +30,59 @@ You'll need to download one of the `datadog-agent*_amd64.deb` package in this di
 You can then build the image using `docker build -t datadog/agent:master .`
 
 To build the jmx variant, add `--build-arg WITH_JMX=true` to the build command
+
+## How to activate log collection
+
+The Datadog Agent can collect logs from containers starting at the version 6. Two installations are possible:
+ 
+- on the host: where the agent is external to the Docker environment 
+- or by deploying its containerized version in the Docker environment
+
+### Setup
+
+First let’s create two directories on the host that we will later mount on the containerized agent:
+
+- `/opt/datadog-agent/run`: to make sure we do not lose any logs from containers during restarts or network issues we store on the host the last line that was collected for each container in this directory
+- ` /opt/datadog-agent/conf.d`: this is where you will provide your integration instructions. Any configuration file added there will automatically be picked up by the containerized agent when restarted. For more information about this check [here](https://github.com/DataDog/docker-dd-agent#enabling-integrations).
+
+To  run a Docker container which embeds the Datadog Agent to monitor your host use the following command:
+
+```
+docker run -d --name dd-agent -h `hostname` -e DD_API_KEY=<YOUR_API_KEY>  -e DD_LOG_ENABLED=true -e SD_BACKEND=docker -v /var/run/docker.sock:/var/run/docker.sock:ro -v /proc/:/host/proc/:ro -v /opt/datadog-agent/run:/opt/datadog-agent/run:rw -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro -v /opt/datadog-agent/conf.d:/conf.d:ro datadog/agent:6.0.0-beta.6
+```
+
+*Important notes*: 
+
+- The Docker integration is enabled by default, as well as [autodiscovery](https://docs.datadoghq.com/guides/servicediscovery/) in auto config mode (remove the `SD_BACKEND` variable to disable it).
+
+- For this example we used the image 6.0.0-beta.6 of the agent. You can find [here](https://hub.docker.com/r/datadog/agent/tags/) the latest available images for agent 6 and we encourage you to always pick the latest version
```

Comment:
Starting with `beta7`, to be out in a few hour, we'll be providing the `latest` tag, could you update accordingly?

Reply:
@xvello Thanks for the update, that's an awesome news. i'll make the update

Reply:
👍  updated